test/src/main/java/org/axonframework/test/matchers/FieldFilter.java

```diff
@@ -32,5 +30,9 @@ public interface FieldFilter {
      * @param field The field to evaluate
      * @return <code>true</code> when the field should be processed, otherwise <code>false</code>
      */
-    boolean accept(Field field);
+    boolean accept(String field);
+
+    default String getField() {
+      return null;
```

Comment:
Does it still make sense to have a default implementation returning `null`? It was done in order not to modify all the `FieldFilter` implementations.